dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -351,15 +351,12 @@ public static List<Class<?>> getStartupRunOnceTaskClasses() {
 		.add(Task241009CreatePostgresJobQueueTables.class)
 		.add(Task241013RemoveFullPathLcColumnFromIdentifier.class)
 		.add(Task241014AddTemplateValueOnContentletIndex.class)
+		.add(Task241009ReplaceLanguagesWithLocalesPortlet.class)
```

Comment:
This task won't run with that name... The number in the name needs to be higher than the previous to get executed